ASHRAE 55 Tables/mrt inputs

```diff
@@ -0,0 +1,192 @@
+// default values
+mrt.room.depth = 30
+mrt.room.width = 15
```

Comment:
What's this file for?

Reply:
was to generate the results for the ASHRAE tables. I have removed it.